pkg/apis/extensions/v1beta1/generated.proto

```diff
@@ -786,6 +786,13 @@ message PodSecurityPolicySpec {
   // If set to false the container may run with a read only root file system if it wishes but it
   // will not be forced to.
   optional bool readOnlyRootFilesystem = 14;
+
+  // SeccompProfiles lists the allowed profiles that may be set for the pod or
+  // container's seccomp annotations.  An unset (nil) or empty value means that no profiles may
+  // be specifid by the pod or container.	The wildcard '*' may be used to allow all profiles.  When
+  // used to generate a value for a pod the first non-wildcard profile will be used as
+  // the default.
+  repeated string seccompProfiles = 15;
```

Comment:
this is confusing re: the concept of having multiple profiles. People will need to understand that you can only further secure by adding additional profiles, you cannot negate behavior. This will need to be documented heavily.